modules/virt-about-cpu-and-memory-quota-namespace.adoc

```diff
@@ -9,9 +9,5 @@ A _resource quota_, defined by the `ResourceQuota` object, imposes restrictions
 a namespace that limit the total amount of compute resources that can be
 consumed by resources within that namespace.
 
-The `CDIConfig` object defines the user configuration for the Containerized Data Importer (CDI). The CPU and
-memory request and limit values for the `CDIConfig` object are set to a default
-value of 0.
-This ensures that pods created by CDI that make no compute resource requirements
-are given the default values and are allowed to run in a namespace that is restricted
-with a quota.
+The `CDI` object defines the user configuration for the Containerized Data Importer (CDI). The CPU and memory request and limit values for the `CDI` object are set to a default value of 0.
+This ensures that pods created by CDI that make no compute resource requirements are given the default values and are allowed to run in a namespace that is restricted with a quota.
```

Comment:
"pods...that make no compute resource requirements"

The "make" gives me pause here. I know it's only showing up in this PR because you unwrapped the lines, though.

Reply:
Ah yeah, I see what you mean.
Update to "...by CDI that do not specify computer resource requirements..."
@sjhala-ccs Can you please confirm that this change doesn't alter the original meaning?

Reply:
@aburdenthehand The rewording makes sense and does not alter the original meaning. Thanks @ousleyp for catching this!